tests/unit/broccoli/default-packager/helpers.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const DEFAULT_APPLICATION_SOURCE = {
+  app: {
```

Comment:
this has gotten extremely verbose, maybe we can find a better way to work with fixtures.

Reply:
true that! we all agree that `ember-app` is trying to do too much, the key here would be to draw a clear boundary between them